book/scripts.md

```diff
@@ -173,7 +173,7 @@ class JSContext:
 ```
 
 DukPy's `JSInterpreter` object stores the values of all the JavaScript
-variables and lets us run multiple JavaScript snippets and share
+variables, and lets us run multiple JavaScript snippets and share
```

Comment:
I have no idea whether the comma should be there, let's roll the dice and see what the copyeditor says :)